Sources/DistributedActorsTestKit/ActorTestKit.swift

```diff
@@ -81,7 +81,8 @@ extension ActorTestKit {
             name = ActorTestProbe<Message>.naming.makeName(&self.namingContext)
         }
 
-        return ActorTestProbe({ probeBehavior in
+        return ActorTestProbe(
+            { probeBehavior in
```

Comment:
hate it when swift format messes with those... is there some setting to prevent it?